fdpa-resolutions.html

```diff
@@ -86,5 +28,159 @@ <h2>Fair Districts PA - Local Resolutions</h2>
         &nbsp; | &nbsp; Click a county or municipality for more info &nbsp; | &nbsp;
         <a href="https://tinytake.com/how-to-take-a-screenshot/">Help with screen shots</a></p>
     <div id="map-canvas"></div>
+    <script src="https://www.promisejs.org/polyfills/promise-7.0.4.min.js">
+    </script>
+    <script type="text/javascript">
+      var map;
+      var layer_0;
+      var layer_1;
+      function initialize() {
+        map = new google.maps.Map(document.getElementById('map-canvas'), {
+          center: new google.maps.LatLng(40.99058069667954, -77.62118299218751),
+          zoom: 8,
+          zoomControl: true,
+              zoomControlOptions: {
+              position: google.maps.ControlPosition.TOP_LEFT
+          },
+          fullscreenControl: true,
+          streetViewControl: false,
+          mapTypeControl: false
+        });
+        var style = [
+          {
+            featureType: 'all',
+            elementType: 'all',
+            stylers: [
+              { saturation: -85 }
+            ]
+          },
+          {
+            featureType: 'poi',
+            elementType: 'all',
+            stylers: [
+              { visibility: 'off' }
+            ]
+          }
+        ];
+        var styledMapType = new google.maps.StyledMapType(style, {
+          map: map,
+          name: 'Styled Map'
+        });
+        map.mapTypes.set('map-style', styledMapType);
+        map.setMapTypeId('map-style');
+        layer_0 = new google.maps.FusionTablesLayer({
+          query: {
+            select: "col12>>0",
+            from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
+            where: " col3>>0 = 'County'"
+          },
+          map: map,
+          styleId: 3,
+          templateId: 4
+        });
+        layer_1 = new google.maps.FusionTablesLayer({
+          query: {
+            select: "col16>>0",
+            from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
+            where: " col3>>0 in ('Borough', 'City', 'Town', 'Township (1st)', 'Township (2nd)', 'Organization') and col5>>0 in ('In work', 'Materials sent', 'Passed')"
+          },
+          map: map,
+          styleId: 4,
+          templateId: 5
+        });
+      }
+      (function() {
+        var apiKey = {
+          amyprime: 'AIzaSyBxdPyKNXkcFbFPs5I_e-xqGCUPGVuyabs',
+          devel: 'AIzaSyBz3CQw1aK0S9UHwxKq-YZtIeqdcOby-Jc'
+        };
+        var locationUrl = new URL(window.location.href);
+
+        addGoogleMaps();
+
+        var statuses = ['passed', 'inWork'];
+        var countElIds = {
+          fallbackCount: 'fall-back-count',
+          currentCount: 'current-count'
+        };
+        var windowPromise =

```

Comment:
All this promise stuff is obvious to me, but if it isn't clear to the reviewer, it may be worth adding comments.

Reply:
I don't follow it 100%, but it looks like I could if I tried harder, so I'm not sure comments are necessary.  And I think I'd mostly be spending that time thinking about the async nature than the promise syntax.  Maybe it would be good to just turn your note "Promises are the best way to join multiple async requests" above into a real comment in the code.

Reply:
Okay, the new comments are super awesome.